docs/api/tutorials/get-the-nutriscore.md

```diff
@@ -16,20 +14,12 @@
 
 Please use only the official assets to display the Nutri-Score. You can get v1 logos here: [NutriScore variants](https://drive.google.com/drive/u/1/folders/13SL2hgqYHSLMhYjMze9nYXV9GOdGMBgc)
 
```

Comment:
```suggestion
Those are v1 assets, and although they can be used during the transition period, we recommand using the blue banner version to avoid any ambiguity.
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a.svg
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-b.svg
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-c.svg
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-d.svg
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-e.svg
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-unknown.svg
- https://static.openfoodfacts.org/images/attributes/dist/nutriscore-not-applicable.svg
```